docs/getting-started/what-is-dotnet.md

```diff
@@ -133,18 +139,17 @@ The .NET Framework on the Windows desktop allows you to create console applicati
 
 - Console applications can be compiled to run cross-platform on Windows, the Macintosh, and Linux.
 
-- Console applications can be compiled natively. This combines the benefits of a managed application with the performance of a native C/C++ native application. 
+- Console applications can be compiled natively. This combines the benefits of a managed application with the performance of a native C/C++ application. 
 
 ## Ease of Installation ##
 
 Because .NET Core is not an operating system component, installation does not require administrator privileges, and it does not touch system components like operating system directories or, on Windows systems, the Windows Registry. Installation is as simple as copying some files to a directory on the target computer or compiling the framework natively into the application. 
```

Comment:
Drop because
.NET core is not an OS component:

- installation does not require administrator privileges,
- it does not touch OS  components
- Installation is as simple as copying some files to a directory on the target computer or compiling the framework natively into the application.